.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,26 @@
 ### Additional Information
 _The following information is very important in order to help us to help you. Omission of the following details may delay your support request or receive no attention at all._
+_Keep in mind that the commands we provide to retrieve information are oriented to GNU/Linux Distributions, so you could need to use others if you use s3fs on MacOS or BSD_
 
 #### Version of s3fs being used (s3fs --version)
 _example: 1.00_
 
-#### Version of fuse being used (pkg-config --modversion fuse)
+#### Version of fuse being used (pkg-config --modversion fuse, rpm -qi fuse, dpkg -s fuse)
 _example: 2.9.4_
 
-#### System information (uname -r)
+#### Kernel information (uname -r)
 _command result: uname -r_
 
-#### Distro (cat /etc/issue)
-_command result: cat /etc/issue_
+#### GNU/Linux Distribution, if applicable (cat /etc/os-release)
+_command result: cat /etc/os-release_
 
-#### s3fs command line used (if applicable)
+#### s3fs command line used, if applicable
 ```
 ```
-#### /etc/fstab entry (if applicable): 
+#### /etc/fstab entry, if applicable
 ```
 ```
-#### s3fs syslog messages (grep s3fs /var/log/syslog, or s3fs outputs)  
+#### s3fs syslog messages (grep s3fs /var/log/syslog, journalctl | grep s3fs, or s3fs outputs)
```

Comment:
`/var/log/syslog` is not available anymore for some distributions using systemd, and in those cases journalctl is the replacement.

Reply:
Thanks